aws/templates/commonApplication.ftl

```diff
@@ -211,6 +211,23 @@
     ]
 [/#function]
 
+[#function standardPolicies occurrence ]
+    [#return
+        valueIfContent(
+            credentialsDecryptPermission,
+            occurrence.Configuration.Environment.Sensitive,
+            []
+        ) +
+        s3ReadPermission(operationsBucket, getSettingsFilePrefix(occurrence)) +
+        s3AllPermission(dataBucket, getAppDataFilePrefix(occurrence)) +
+        valueIfContent(
+            s3AllPermission(dataBucket, getAppDataPublicFilePrefix(occurrence)),
+            getAppDataPublicFilePrefix(occurrence),
+            []
+        )
+    ]
+[/#function]
+
```

Comment:
Should cloudwatch be a default permission as well? If everything is going to log to a Cloudwatch prefix then its available if anything can use it.